test/uniq.js

```diff
@@ -0,0 +1,59 @@
+var shell = require('..');
+
+var assert = require('assert'),
+    fs = require('fs');
+
+shell.config.silent = true;
+
+shell.rm('-rf', 'tmp');
+shell.mkdir('tmp');
+
+var result;
+
+result = shell.uniq();
```

Comment:
Please mark this section with:

```
//
// Invalids
//
```

See [here](https://github.com/shelljs/shelljs/blob/master/test/cat.js#L11-13) as an example.